spec/system/visitor_entry/manager_registers_a_vistor_entry_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+describe 'Manager registers a visitor entry' do
+  it 'from condo dashboard' do
+    manager = create :manager
+    condo = create :condo
+
+    login_as manager, scope: :manager
+    visit condo_path condo
+    click_on 'Registrar entrada de visitante'
+
+    expect(page).to have_content 'Registrar nova entrada'
+    expect(current_path).to eq new_condo_visitor_entry_path condo
+  end
+
+  it 'and must be authenticated' do
+    condo = create :condo
+
+    visit new_condo_visitor_entry_path condo
+
+    expect(current_path).to eq new_manager_session_path
+  end
+
+  it 'successfully without reference a unit' do
+    manager = create :manager
+    condo = create :condo
+
+    login_as manager, scope: :manager
+    visit new_condo_visitor_entry_path condo
+    fill_in 'Nome Completo',	with: 'João da silva'
+    fill_in 'RG',	with: '123456789'
+    select '', from: 'Unidade visitada'
+    click_on 'Criar Entrada de Visitante'
+
+    expect(page).to have_content 'Entrada de visitante registrada com sucesso'
+    expect(current_path).to eq condo_visitor_entries_path condo
+    expect(page).to have_content 'João da silva'
+    expect(page).to have_content '123456789'
+    expect(page).to have_content 'Sem unidade referenciada'
+    expect(page).to have_content I18n.l(VisitorEntry.last.created_at, format: :long)
+  end
+
+  it 'successfully with reference to a unit' do
+    manager = create :manager
+    condo = create :condo
+    unit_type = create(:unit_type, condo:)
+    create(:tower, :with_four_units, condo:, name: 'Torre A', unit_types: [unit_type])
+
+    login_as manager, scope: :manager
+    visit new_condo_visitor_entry_path condo
+    fill_in 'Nome Completo',	with: 'João da silva'
+    fill_in 'RG',	with: '123456789'
+    select 'Torre A', from: 'Torre'
+    select '11', from: 'Unidade visitada'
+    click_on 'Criar Entrada de Visitante'
+
+    expect(page).to have_content 'Entrada de visitante registrada com sucesso'
+    expect(current_path).to eq condo_visitor_entries_path condo
+    expect(page).to have_content 'João da silva'
+    expect(page).to have_content '123456789'
+    expect(page).to have_content 'Torre A'
+    expect(page).to have_content '11'
+    expect(page).to have_content I18n.l(VisitorEntry.last.created_at, format: :long)
+  end
+
+  it 'with missing params' do
```

Comment:
Porque não faz a união entre o teste `with missing params` e `identity number must be numbers and letters only` ?
Pelo que notei, a inicialização e as ações estão idênticas, apenas as expectativas finais que estão diferentes.

Reply:
Na realidade, foi um erro. No segundo teste era para preencher com algo que nao fosse letra ou numero. vou arrumar